_layouts/category_index.html

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+---
+{% if page.paginator %}
+
+  <!-- Pagination is active -->
+  {% assign paginator = page.paginator %}
+
+  <br>
+  <a class="tla-link" href="{{site.baseurl}}/blog/category/"><span aria-hidden="true">&larr;</span> All Categories</a>
+
+  <h1>Posts in category: &ldquo;{{ page.title }}&rdquo;</h1>
+
+  <section>
+    <!-- Show post excerpts for the current page -->
+
+    {% for post in paginator.posts limit:paginator.per_page %}
+    <hr class="divider" />
+    <section>
+      <h2 class="title">
+        <a class="tla-link" href="{{post.url}}">{{post.title}}</a>
+        <small>by {{post.author}} <span class="text-normal">on {{ post.date | date: "%b %-d, %Y" }}</span></small>
+      </h2>
+      <p>
+        <!-- TODO: IMPLEMENT TAGS! -->
+        {% comment %}
+        {% if post.tags %}
+            tags:
+            {% for tag in post.tags %}
+                {{tag}}{% if forloop.last == false %}, {% endif %}
+            {% endfor %}
+        {% endif %}
+        {% endcomment %}
+      </p>
+      <blockquote class="post-blockquote">
+        {{post.excerpt}} <a class="tla-link" href={{post.url}}>... read more!</a>
+      </blockquote>
+    </section>
+    {% endfor %}
+  </section>
+  <!-- Show navigation next/previous page links if applicable -->
+  <nav><p>
+    <h2 class="title" style="display: flex; flex-flow: row wrap; align-items: center">
+      <div style="flex:1" class="text-center">
+        {% if paginator.previous_page %}
+        <a class="tla-link" href="{{ paginator.previous_page_path }}"><span aria-hidden="true">&larr;</span> Newer</a>
+        {% endif %}
+      </div>
+
+      <div style="flex:1" class="text-center">
+        {% if paginator.previous_page != nil or paginator.next_page != nil %}
+        <!-- Show page number if we have multiple pages -->
+        <span> 
+          Page <span aria-hidden="true"><br></span>
+          {{paginator.page}} of {{paginator.total_pages}} 
+        </span>
+        {% endif %}
+      </div>
+
+      <div style="flex:1" class="text-center">
```

Comment:
If you could move these styles to their own class in our SCSS to keep the code maintainable it would be phenomenal.